monkestation/code/datums/diseases/advance/symptoms/symptoms.dm

```diff
@@ -0,0 +1,2 @@
+/datum/symptom/proc/OnDeath(datum/disease/advance/advanced_disease)    //What happens when the host dies.
+	return !neutered
```

Comment:
why return this?

Reply:
instead of giving me a vague and honestly terrible review as "why return this?" give some proper guidance, all i would need to learn is two lines of text.
from what i can tell there are plenty of examples of disease procs that return neutered, why? i have no clue but from what i can see in other examples it was to fix some issue.
is that relevant here? i dont know!

Reply:
this passes neutered state along, if the symptom is neutered, onDeath should obviously not trigger its effect.
there are many ways to write this...
and in our code we have multiple styles(same with tg)
this could be changed to either 
    if(neutered)
        return FALSE
    return TRUE
    
  or
  
  . = ..()
if(!.)
    return .
    
    they all do the same.